James-Lycett/workplace-wellness
# Register / New Employee stuff (issue #51)

To Test:
- Create a new user via the /register page
- Make sure that user is in the employees table as seen from an admin dashboard, and that all the appropriate table data is there

Changes:
- Removed checkboxes from employee and activity tables
- Updated table columns to only include the data input on account creation (e.g., "occupation" rather than "sleep_hours")
- Updated CreateAccount form (on /register page) to match the "add new employee" form (on /dashboard)
- Removed CreateUserAccount.jsx and CreateAdminAccount.jsx, they weren't even being used
